File: C++/2. Conditionals and Loops/1.cpp
// Conditional Statements
#include <iostream>
using namespace std;

int main()
{
    /*
        if (condition) {
            statements
        }
        else if {
            statements
        }
        else {
            statements
        }
    */
    int a = 55, b = 55;
    if (a > b)
        cout << "a is greater than b" << endl; // if it's only one statement, "{}" are not necessary
    else if (a < b)
        cout << "b is greater than a" << endl;
    else
        cout << "a is equal b" << endl;

    /*
            Logical Operators
        && - logical AND
        || - logical OR
        ! - logical NOT
    */
    int n = 42;
    if (n > 0 && n <= 100 || !(n > 25 && n < 75))
        cout << "Complex condition" << endl;

    return 0;
}

File: Java/2. Conditionals and Loops/1.java
class ConditionalStatements {
    /*
     * Conditional statements are used to perform different actions based on
     * different conditions.
     * 
     * Signs:
     * < - less than
     * > - greater than
     * != - not equal to
     * == - equal to
     * <= - less than or equal to
     * >= - greater than or equal to
     * 
     * Logical Operators:
     * && - AND
     * || - OR
     * ! - NOT
     */
    public static void main(String[] args) {
        int age = 25;
        if (age <= 0) {
            System.out.println("Error");
        } else if (age <= 16) {
            System.out.println("Too Young");
        } else if (age < 100) {
            if (age > 60)
                System.out.println("Welcome, boomer!");
            else
                System.out.println("Welcome!");
        } else {
            System.out.println("Really?");
        }
    }
}
